### DISCUSSION TITLE
Fenced codeblocks retruns only pre and not code

### DISCUSSION THREAD

**Discussion by kobaidze3** [markdoc/markdoc#514] [Q&A] [answered]
<img width="284" alt="image" src="https://github.com/markdoc/markdoc/assets/169984420/8a75a4a1-b72b-44da-9b3d-af20481cd1fa">
Why is this happening? here's the markdown I pass 
```<div> test123 </div>```

**Comment by mfix-stripe** [comment, marked as answer]
Hi @kobaidze3, that is how the tag is designed: https://www.w3schools.com/tags/tag_pre.asp

You can update the transform or render functions of the `fence` node (https://github.com/markdoc/markdoc/blob/main/src/schema.ts#L59) to add an additional `code` tag if you would like.
